### DISCUSSION TITLE
Push to scalar list - only when value doesn't already exist

### DISCUSSION THREAD

**Discussion by jamezening** [prisma/prisma#13014] [Q&A] [answered]
https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#push

Is there a way to push to a scalar list and maintain that scalar list as a non-duplicate set-like data structure?

Thank you for any help!

**Comment by austincrim** [comment, marked as answer]
Hey @vanilla-james 👋 ,

There is no out of the box way to do this. You would need to manually perform the duplicate check in your application logic. You may open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) though!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
